lib/wallet/api/http/Cardano/Wallet/Api/Http/Shelley/Server.hs

```diff
@@ -1780,7 +1783,7 @@ selectCoins ctx@ApiLayer {..} argGenChange (ApiT walletId) body = do
         withdrawal <-
             body ^. #withdrawal
                 & maybe (pure NoWithdrawal)
-                    (shelleyOnlyMkWithdrawal @s @k @n netLayer txLayer db era)
+                    (shelleyOnlyMkWithdrawal @s @k @n netLayer (txWitnessTagFor @k) db era)
```

Comment:
Hi @Unisay: would it be okay to break up this long line? (Longer than the 80 character column limit.)

Perhaps you could write:
```suggestion
                    (shelleyOnlyMkWithdrawal @s @k @n
                        netLayer (txWitnessTagFor @k) db era)
```